packages/react/src/primitives/Tabs/TabsContainer.tsx

```diff
@@ -20,6 +21,8 @@ const TabsContainerPrimitive: Primitive<TabsProps, 'div'> = (
   }: BaseTabsProps,
   ref
 ) => {
+  const groupId = useStableId();
+  const whitespaceValue = '-'; // used to create valid HTML IDs
```

Comment:
Since this never changes, it doesn't need to be declared inside of the component and should be moved outside of the component (near the top of file, under imports)